CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+**Features**:
+
+**Internal**:
+
+- Collect exclusive_time_light metrics for cache.\* spans ([#3466]https://github.com/getsentry/relay/pull/3466)
```

Comment:
the format is incorrect, should be

([#3466](https://github.com/getsentry/relay/pull/3466))

Reply:
Thanks!